doc/debugging.md

```diff
@@ -4,7 +4,7 @@ Once you have set up a `launch.json` file with default configurations, you can m
 
 ![React Native launch configuration file](../images/launch-config.png)
 
-For example, you can modify the `target` field to specify the simulator you want to target for iOS debugging.
+For example, you can modify the `target` field to specify the simulator you want to target for iOS debugging or the device is as output from `adb devices`.
```

Comment:
@lrkwz thanks for your suggestion, but I would change this to "or the device using the one from output of `adb devices`"  to make it more clear to users. If you have any doubts/concerns please let me know.

Reply:
@lrkwz no problem.